pkg/solana/chain_test.go

```diff
@@ -152,9 +210,12 @@ func TestSolanaChain_VerifiedClient(t *testing.T) {
 
 	ch := solcfg.Chain{}
 	ch.SetDefaults()
+	mn := solcfg.MultiNodeConfig{}
```

Comment:
Do we need multiNode in this test? 
It seems to be focused `verifiedClient()`  that does not interact with MultiNode

Reply:
We don't activate MultiNode here, just need it to be defined.  I will use `cfg.SetDefaults()` instead!